p/pep517/pep517_ubi_8.4.sh

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : pep517
+# Version       : v0.11.0
```

Comment:
just one question @gururajrkatti - is this the full list of versions that were validated? If not, we might want to specify all, rest looks good

Reply:
Hi @seth-priya, added the other tested version to the header. Pl. check